src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final Integer HOURS_OF_WORK_INDEX = 2;
+    private static final Integer SALARY_PER_HOUR_INDEX = 3;
+    private static final Integer POSITION_OF_DATE_IN_STRING = 0;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate startCounting = LocalDate.parse(dateFrom, formatter1);
+        LocalDate endCounting = LocalDate.parse(dateTo, formatter1);
```

Comment:
```suggestion
        LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
        LocalDate endDate = LocalDate.parse(dateTo, FORMATTER);
```
make formatter constant